crunch/headings-and-subtotals.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
This is super minor, and might not impact anything at all, but is there any reason not to follow the convention of other files and rename this to `crunch_headings-and-subtotals.md`? I find the mix of `_` and `-` odd, but I see it on each of the other pages.

Reply:
Whoa totally missed that!